nuget/build/AspectInjector.targets

```diff
@@ -20,10 +20,11 @@
   </PropertyGroup>
 
   <PropertyGroup Condition="'$(AspectInjector_Location)' == ''">
-    <AspectInjector_Location Condition="$([MSBuild]::IsOSPlatform('OSX'))">$(MSBuildThisFileDirectory)../../build/_bin/osx-x64/AspectInjector</AspectInjector_Location>
-    <AspectInjector_Location Condition="$([MSBuild]::IsOSPlatform('Windows'))">$(MSBuildThisFileDirectory)../../build/_bin/win-x64/AspectInjector.exe</AspectInjector_Location>
-    <AspectInjector_Location Condition="$([MSBuild]::IsOSPlatform('Linux'))">$(MSBuildThisFileDirectory)../../build/_bin/linux-x64/AspectInjector</AspectInjector_Location>
-    <AspectInjector_Location Condition="$([MSBuild]::IsOSPlatform('Linux')) AND Exists('/lib/libc.musl-x86_64.so.1')">$(MSBuildThisFileDirectory)../../build/_bin/linux-musl-x64/AspectInjector</AspectInjector_Location>    
+    <AspectInjector_Location Condition="'$(RuntimeIdentifier)' == 'osx-x64'">$(MSBuildThisFileDirectory)../../build/_bin/osx-x64/AspectInjector</AspectInjector_Location>
```

Comment:
it isn't going to work. RuntimeIdentifier is a target platform, but here we need to figure what's out host platform so we can use correct executable

Reply:
I see. Thanks for your time. I took some time yesterday evening to test some scenarios and the current commit should contain a fix that could work.